cdap-watchdog/src/main/java/co/cask/cdap/logging/gateway/handlers/LogDataOffsetCallback.java

```diff
@@ -44,7 +44,7 @@ public Object encodeSend(LogEvent logEvent) {
     String className = "";
     String simpleClassName = "";
     int lineNumber = 0;
-    if (stackTraceElements.length > 0) {
+    if (stackTraceElements != null && stackTraceElements.length > 0) {
       StackTraceElement first = stackTraceElements[0];
       className = first.getClassName();
       if (className.lastIndexOf('.') >= 0) {
```

Comment:
Should `simpleClassName` be equal to `className` if `className` does not contain a `.`?